scripts/quick-start-koku.sh

```
@@ -210,48 +238,38 @@ cat <<- EEOOFF
 
 EEOOFF
 }
 
 # Tear down the koku virtualenv and stop/remove containers from a previous run.
-# Always destructive — removes volumes and all database data.
+# Always destructive — removes volumes (Postgres, S4/Hive warehouse) and all data.
+# Leaving the koku-s4-data volume causes Trino HIVE_PATH_ALREADY_EXISTS on the next
+# OCP summary (metastore empty, S3 path still present) so has_data never becomes true.
 clean()
 {
   cd $KOKU_DIR
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)